README.md

```diff
@@ -76,138 +72,88 @@ So your options are:
 * `askTimeoutInMilliSec`: Number of milliseconds to await an Akka response on publishing a message. This is used when we want to store the delivery tag of the message to handle confirmations and resending messages which lack confirmation. 
 * `exchanges`: The list of exchanges you would like to use. Built in exchanges (amq.direct, amq.topic, etc.) are included by default, you don't have to add them here. The index of the exchange config will be the name of the exchange. With every exchange, you can configure:
   * `type`: The type of exchange (direct, topic, fanout, headers) 
-  * `deadLetterExchange`: The name of dead letter exchange for the exchange. You have to configure that here also, or you can use on of the built in exchanges. 
+  * `deadLetterExchange`: The name of dead letter exchange for the exchange. You have to configure that here also, or you can use on of the built in exchanges.
+  * `atLeastOnce`: Delivery guarantee used for this exchange. Default value is assumed if this is omitted. It's referred in code as `AtLeastOnceGroup.default`.
+* `builtinAtLeastOnce`: Delivery guarantee used for several predefined exchanges, like `amq.topic`. Default value is assumed if this is omitted. It's referred in code as `AtLeastOnceGroup.default`.
```

Comment:
Documentation here should probably detail what this "group" concept means. That you can implement various message stores, and you can group exchanges into delivery guarantee groups based on what delivery guarantee the message store provides. Or something like this. Examples would be useful too. And shouldn't we call these config keys `atLeastOnceGroup` and `builtinAtLeastOnceGroup` respectively? Make sure to also rename occurrence in `AmqpConfiguration.scala`.